_includes/examples/carriers/ups/ship_webservice/misc/access_point_delivery.html

```diff
@@ -17,19 +17,24 @@ <h4 id="ship-webservice---access-point-delivery" class="text-blue">
     <code><em>to</em></code> attribute
   </li>
   <li>
-    you have to provide the exact address of the UPS access point using the
-    <code><em>drop_off_point</em></code> attribute
+    <code><em>to.first_name</em></code> is manadatory
   </li>
   <li>
-    the attribute <code><em>drop_off_point.type</em></code> has to be
-    <code><em>parcel_shop</em></code>
+    <code><em>to.last_name</em></code> is manadatory
+  </li>
+  <li>
+    <code><em>to.email</em></code> is manadatory
   </li>
   <li>
-    you also have to use the additional service
-    <code><em>ups_access_point_notification</em></code>
-    (<a data-controller="toggle-element"
-      data-target="#ship_webservice_access_point_delivery_togglebox" aria-expanded="false"
-      aria-controls="collapseExample" class="cursor-pointer">see example</a>)
+    <code><em>drop_off_point.drop_off_point_id</em></code> is manadatory and has to be 9 characters long (can be determined through our
+    <a style="white-space: nowrap;" href="{{ site.baseurl }}/swagger-ui/#/default/get_pickup_dropoff_locations">
+      pudo provider
+    </a>
+    )
```

Comment:
This produces an unwanted whitespace which I do not know how to fix, like this:
(can be determined through our [pudo provicer](https://developers.shipcloud.io/swagger-ui/#/default/get_pickup_dropoff_locations) )

Reply:
Could you put everything in one line? Would that fix that?
```
<a style="white-space: nowrap;" href="{{ site.baseurl }}/swagger-ui/#/default/get_pickup_dropoff_locations">pudo provider</a>)

Reply:
It did fix the problem, thank you.